image_data_guide/04a_builtin_training.ipynb

```diff
@@ -0,0 +1,487 @@
+      "\u001b[34m[10/05/2020 05:45:20 INFO 140481505945408] Epoch[1] Validation-accuracy=0.587500\u001b[0m\n",
+      "\u001b[34m[10/05/2020 05:45:20 INFO 140481505945408] Storing the best model with validation accuracy: 0.587500\u001b[0m\n",
+      "\u001b[34m[10/05/2020 05:45:20 INFO 140481505945408] Saved checkpoint to \"/opt/ml/model/image-classification-0002.params\"\u001b[0m\n",
+      "\u001b[34m[10/05/2020 05:45:22 INFO 140481505945408] Epoch[2] Batch [20]#011Speed: 641.844 samples/sec#011accuracy=0.717262\u001b[0m\n",
+      "\u001b[34m[10/05/2020 05:45:24 INFO 140481505945408] Epoch[2] Train-accuracy=0.731618\u001b[0m\n",
+      "\u001b[34m[10/05/2020 05:45:24 INFO 140481505945408] Epoch[2] Time cost=3.194\u001b[0m\n",
+      "\u001b[34m[10/05/2020 05:45:24 INFO 140481505945408] Epoch[2] Validation-accuracy=0.640625\u001b[0m\n",
+      "\u001b[34m[10/05/2020 05:45:24 INFO 140481505945408] Storing the best model with validation accuracy: 0.640625\u001b[0m\n",
+      "\u001b[34m[10/05/2020 05:45:24 INFO 140481505945408] Saved checkpoint to \"/opt/ml/model/image-classification-0003.params\"\u001b[0m\n",
+      "\u001b[34m[10/05/2020 05:45:26 INFO 140481505945408] Epoch[3] Batch [20]#011Speed: 636.940 samples/sec#011accuracy=0.775298\u001b[0m\n",
+      "\u001b[34m[10/05/2020 05:45:27 INFO 140481505945408] Epoch[3] Train-accuracy=0.778493\u001b[0m\n",
+      "\u001b[34m[10/05/2020 05:45:27 INFO 140481505945408] Epoch[3] Time cost=3.193\u001b[0m\n",
+      "\u001b[34m[10/05/2020 05:45:28 INFO 140481505945408] Epoch[3] Validation-accuracy=0.668750\u001b[0m\n",
+      "\u001b[34m[10/05/2020 05:45:28 INFO 140481505945408] Storing the best model with validation accuracy: 0.668750\u001b[0m\n",
+      "\u001b[34m[10/05/2020 05:45:28 INFO 140481505945408] Saved checkpoint to \"/opt/ml/model/image-classification-0004.params\"\u001b[0m\n",
+      "\u001b[34m[10/05/2020 05:45:30 INFO 140481505945408] Epoch[4] Batch [20]#011Speed: 630.140 samples/sec#011accuracy=0.796875\u001b[0m\n",
+      "\u001b[34m[10/05/2020 05:45:31 INFO 140481505945408] Epoch[4] Train-accuracy=0.795496\u001b[0m\n",
+      "\u001b[34m[10/05/2020 05:45:31 INFO 140481505945408] Epoch[4] Time cost=3.200\u001b[0m\n",
+      "\u001b[34m[10/05/2020 05:45:31 INFO 140481505945408] Epoch[4] Validation-accuracy=0.687500\u001b[0m\n",
+      "\u001b[34m[10/05/2020 05:45:32 INFO 140481505945408] Storing the best model with validation accuracy: 0.687500\u001b[0m\n",
+      "\u001b[34m[10/05/2020 05:45:32 INFO 140481505945408] Saved checkpoint to \"/opt/ml/model/image-classification-0005.params\"\u001b[0m\n",
+      "\n",
+      "2020-10-05 05:45:45 Completed - Training job completed\n",
+      "Training seconds: 133\n",
+      "Billable seconds: 133\n"
+     ]
+    }
+   ],
+   "source": [
+    "algorithm.fit(inputs=data_channels, logs=True)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "<pre>\n",
+    "\n",
+    "</pre>"
+   ]
+  },
+  {
```

Comment:
Is there a way to plot the val. accuracy and/or the training loss per epoch? I think there are other tools like CloudWatch, but I wonder if there is a way we can plot this within a notebook and not force folks to click elsewhere.

Reply:
Cool, I will look into a simple way of getting this data